web/docs/data-model/crud.md

```diff
@@ -149,6 +149,10 @@ We also overrode the `create` operation with a custom implementation. This means
 
 ### Our Custom `create` Operation
 
+We need a custom `create` operation because we want to make sure that the task is connected to the user creating it.
+Automatic CRUD doesn't support this by default (yet!).
```

Comment:
```suggestion
Automatic CRUD doesn't yet support this by default.
```

Reply:
I think this formulation was yours and I actually liked it, but yeah, the suggestion is more normal.